core/node/rpc/multi.go

```diff
@@ -218,18 +218,22 @@ func GetRiverNetworkStatus(
 	if err != nil {
 		return nil, err
 	}
+	defer http11client.CloseIdleConnections()
```

Comment:
With DisableKeepAlives: true by default for Http11Client, this line is now redundant and can be removed in favor of the transport setting.